packages/cli/src/webhooks/webhook-form-data.ts

```diff
@@ -0,0 +1,44 @@
+import formidable from 'formidable';
+import type { IncomingMessage } from 'node:http';
+
+type IncomingMessageWithEncoding = IncomingMessage & { encoding: string };
+
+const normalizeFormData = <T>(values: Record<string, T | T[]>) => {
+	for (const key in values) {
+		const value = values[key];
+		if (Array.isArray(value) && value.length === 1) {
+			values[key] = value[0];
+		}
+	}
+};
+
+/**
+ * Creates a function that parses the multipart form data into the request's `body` property
+ */
+export const createMultiFormDataParser = (maxFormDataSizeInMb: number) => {
+	const parseMultipartFormData = async (
```

Comment:
is there a reason to assign this to a const, instead of just returning here?

Reply:
Not really, besides giving a name to the function